### DISCUSSION TITLE
When I copied the example from the standard library, I couldn't simulate it.

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10740] [Q&A] [answered]
The following error is an example copied from the standard library of OMEdit into a package I created, but I'm encountering errors during execution. What could be the cause of this issue, and how can I resolve it?
(The standard library can simulate Buildings successfully.)

/usr/bin/sh: clang -I. -o AbsorptionIndirectSteamVaryingLoad.exe AbsorptionIndirectSteamVaryingLoad.o AbsorptionIndirectSteamVaryingLoad_functions.o AbsorptionIndirectSteamVaryingLoad_records.o AbsorptionIndirectSteamVaryingLoad_01exo.o AbsorptionIndirectSteamVaryingLoad_02nls.o AbsorptionIndirectSteamVaryingLoad_03lsy.o AbsorptionIndirectSteamVaryingLoad_04set.o AbsorptionIndirectSteamVaryingLoad_05evt.o AbsorptionIndirectSteamVaryingLoad_06inz.o AbsorptionIndirectSteamVaryingLoad_07dly.o AbsorptionIndirectSteamVaryingLoad_08bnd.o AbsorptionIndirectSteamVaryingLoad_09alg.o AbsorptionIndirectSteamVaryingLoad_10asr.o AbsorptionIndirectSteamVaryingLoad_11mix.o AbsorptionIndirectSteamVaryingLoad_12jac.o AbsorptionIndirectSteamVaryingLoad_13opt.o AbsorptionIndirectSteamVaryingLoad_14lnz.o AbsorptionIndirectSteamVaryingLoad_15syn.o AbsorptionIndirectSteamVaryingLoad_16dae.o AbsorptionIndirectSteamVaryingLoad_17inl.o AbsorptionIndirectSteamVaryingLoad_18spd.o "-IC:/Users/4006272/AppData/Roaming/.openmodelica/libraries/Modelica 4.0.0+maint.om/Resources/C-Sources" -I"C:/Program Files/OpenModelica1.21.0-64bit/include/omc/c" -I"C:/Program Files/OpenModelica1.21.0-64bit/include/omc" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=AbsorptionIndirectSteamVaryingLoad -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=3 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0 -L"C:/Users/4006272/Desktop/蟾･蝣ｴVPP/forOMEdit"  "-LC:/Program Files/OpenModelica1.21.0-64bit/bin/" "-LC:/Program Files/OpenModelica1.21.0-64bit/lib//omc" "-LC:/Program Files/OpenModelica1.21.0-64bit/lib/" "-LC:/Users/4006272/AppData/Roaming/.openmodelica/binaries/Modelica" "-LC:/Users/4006272/AppData/Roaming/.openmodelica/libraries/Modelica 4.0.0+maint.om/Resources/Library/mingw64" "-LC:/Users/4006272/AppData/Roaming/.openmodelica/libraries/Modelica 4.0.0+maint.om/Resources/Library/win64" "-LC:/Users/4006272/AppData/Roaming/.openmodelica/libraries/Modelica 4.0.0+maint.om/Resources/Library" -lModelicaExternalC  -Os -Wno-parentheses-equality -falign-functions -mstackrealign -msse2 -mfpmath=sse     "-IC:/Users/4006272/AppData/Roaming/.openmodelica/libraries/Modelica 4.0.0+maint.om/Resources/C-Sources" -I"C:/Program Files/OpenModelica1.21.0-64bit/include/omc/c" -I"C:/Program Files/OpenModelica1.21.0-64bit/include/omc" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=AbsorptionIndirectSteamVaryingLoad -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=3 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0 -L"C:/Program Files/OpenModelica1.21.0-64bit/lib//omc" -L"C:/Program Files/OpenModelica1.21.0-64bit/lib" -Wl,--stack,16777216,-rpath,"C:/Program Files/OpenModelica1.21.0-64bit/lib//omc" -L"C:/Program Files/OpenModelica1.21.0-64bit/bin" -Wl,-rpath,"C:/Program Files/OpenModelica1.21.0-64bit/lib"  -fopenmp -Wl,-Bstatic -lregex -ltre -lintl -liconv -lexpat -lpthread -loleaut32 -limagehlp -lhdf5 -lz -lszip -Wl,-Bdynamic -lSimulationRuntimeC -Wl,-Bdynamic -lomcgc -lopenblas -Wl,-Bdynamic  -lstdc++ -Wl,-Bdynamic  : No such file or directory
mingw32-make: *** [AbsorptionIndirectSteamVaryingLoad.makefile:37: omc_main_target] Error 127
mingw32-make: Leaving directory 'C:/Users/4006272/AppData/Local/Temp/OPENMO~1/OMEdit/PACKAG~3.ABS'
Compilation process failed. Exited with code 0x00000002.

**Comment by casella** [comment, marked as answer]
The Duplicate function doesn't (yet) work well. This is known issue #5346, we're working on it 😅

**Reply by ZackeyZAKI** [reply]
Thank you very much!

The reason for that error seems to be because some chinese characters were used in the absolute path. When the characters in the absolute path were changed to alphanumeric characters, the compilation and simulation were successful.

**Comment by casella** [comment]
This issue should be mostly resolved now, see #8777, #8507, #2939.
